README.md

```diff
@@ -124,17 +123,17 @@ $ sudo apt-get install python-rosdep python-catkin-tools
 
 For Create 2 (Roomba 600/700 series):
 ``` bash
-$ roslaunch create_bringup create_2.launch
+$ ros2 launch create_bringup create_2.xml
```

Comment:
I think we should rename the launch files so they have the extension `*.launch.xml`. This seems to make tab-completion work when executing `ros2 launch create_bringup <tab>`.

Reply:
Please disregard my previous comment regarding launch files. I've renamed them to `.launch` and it's working as expected.